cdap-unit-test/src/test/java/co/cask/cdap/spark/SparkTestRun.java

```diff
@@ -294,6 +302,50 @@ public void testScalaSparkWithObjectStore() throws Exception {
   }
 
   @Test
+  public void testScalaSparkCrossNSStream() throws Exception {
```

Comment:
edit: my bad, just checked that both of them are Scala. Do we have Java tests too?